requirements.txt

```diff
@@ -1,2 +1 @@
-httplib2==0.10.3
```

Comment:
should these be removed?

Reply:
No, but as you write it in the setup.py, you can avoid to repeat yourself with the -e .
Here in the requirements.txt, add only the dev dependencies, like pytest for instance.

I suggest you to look at [poetry](https://python-poetry.org/) to really ease the packaging of your library.